targets/TARGET_TT/TARGET_TT_M4G9/Periph_Driver/inc/txz_cg.h

```diff
@@ -0,0 +1,113 @@
+/** 
+ *******************************************************************************
+ * @file    txz_cg.h
+ * @brief   This file provides all the functions prototypes for CG driver.
+ * @version V1.0.0.0
+ * $Date:: 2018-04-02 19:31:41 #$
+ * 
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * (C)Copyright TOSHIBA MICROELECTRONICS CORPORATION 2017 All rights reserved
+ *******************************************************************************
+ */
+/*------------------------------------------------------------------------------*/
+/*  Define to prevent recursive inclusion                                       */
+/*------------------------------------------------------------------------------*/
+#ifndef __CG_H
+#define __CG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*------------------------------------------------------------------------------*/
+/*  Includes                                                                    */
+/*------------------------------------------------------------------------------*/
+#include "txz_driver_def.h"
+
+/**
+ *  @addtogroup Periph_Driver
+ *  @{
+ */
+
+/** 
+ *  @defgroup CG CG
+ *  @brief    CG Driver.
+ *  @{
+ */
+
+/*------------------------------------------------------------------------------*/
+/*  Macro Definition                                                            */
+/*------------------------------------------------------------------------------*/
+/** 
+ *  @defgroup CG_Exported_define CG Exported Define
+ *  @{
+ */
+/* no define */
+/**
```

Comment:
@gaoshaohhua : I see a lot of these sections in many files. Are these files auto generated from any tool? If the commented sections are not required, maybe it will be nice to remove them if not needed? Just to maintain consistency.?